csharp/src/ProtocolBuffers/Google.Protobuf.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Google.Protobuf</id>
+    <title>Google Protocol Buffers C#</title>
+    <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
+    <description>See project site for more info.</description>
+    <version>3.0.0-experimental</version>
```

Comment:
While annoying, the pre-release "viral" nature absolutely makes sense. I think you should either make the version *empty* so that it's invalid in source control, or put it to a version which wouldn't be too bad if you published it.

Possibly 3.0.0-beta* would be okay, as "invalid, but indicating the expected next release number"... (I'm assuming * is invalid in a version number. I haven't checked, but it would be pathologically nasty, at least.)